Adding platform element under featureManager (#313)
OpenLiberty/docs-generated: modules/reference/pages/config/featureManager.adoc

|===
|Name|Type|Default|Description

|+platform+

|string +
This is specified as a child element rather than as an XML attribute.

|

> |+The platform specifies a programming model version; for example "jakartaee-10". The platform determines the version of features for which no version is specified.+

Reviewer: ```suggestion
|+The platform specifies a programming model version, for example jakartaee-10. The platform version determines the version of for any enabled versionless features that are included in the specified platform.+
```

Author: This was copied directly from metatype.xml/property files.  so I can change it here but need to keep coordinated in source as well....

> |+The platform specifies a programming model version, for example jakartaee-10. The platform version determines the version of for any enabled versionless features that are included in the specified platform.+

Reviewer: ```suggestion
|+The platform specifies a programming model version, for example jakartaee-10. The platform version determines the version of any enabled versionless features that are included in the specified platform.+
```
